test/transmuxer.test.js

```diff
@@ -3170,6 +3170,7 @@ QUnit.test('ensures baseMediaDecodeTime for audio is not negative', function() {
     events.push(event);
   });
   audioSegmentStream.track.timelineStartInfo.baseMediaDecodeTime = 10;
```

Comment:
kept this line around because this test actually verifies that this base media decode time is set on one of the boxes.